### ISSUE TITLE
Support Json format

### ISSUE THREAD

**Issue by ngtrian** [GeSHi/geshi-1.0#110]
working with xml, yaml, please add json support

**Comment by cweiske** [comment]
What's your code that is not working?

**Comment by ngtrian** [comment]
This is my code
```json
{
  "short_name": "Maps",
  "name": "Google Maps",
  "icons": [
    {
      "src": "/images/icons-192.png",
      "type": "image/png",
      "sizes": "192x192"
    },
    {
      "src": "/images/icons-512.png",
      "type": "image/png",
      "sizes": "512x512"
    }
  ],
  "start_url": "/maps/?source=pwa",
  "background_color": "#3367D6",
  "display": "standalone",
  "scope": "/maps/",
  "theme_color": "#3367D6"
}
```

**Comment by cweiske** [comment]
I mean your PHP code.

**Comment by ngtrian** [comment]
Json also not list support on homepage GeSHi

**Comment by ngtrian** [comment]
No, if set syntax javascript for json, result in same color. I'm trying in http://qbnz.com.

![image](https://user-images.githubusercontent.com/4327817/46135891-7cc1c700-c270-11e8-879b-7d4e261d2183.png)

Why quickly closed this issue? Instead of discussing further!

**Comment by ngtrian** [comment]
It should highlight as:
![image](https://user-images.githubusercontent.com/4327817/46136046-e641d580-c270-11e8-8785-e66e964da0d9.png)

**Comment by BenBE** [comment]
With enough hackery using COMMENT_REGEXP you could match the string preceeding the colon and matching the string except for the colon (which should be covered by a lookahead only, not as the actual match).
